src/renderer/library/components/opds/PageNavigation.tsx

```diff
@@ -97,6 +97,7 @@ class PageNavigation extends React.Component<IProps, undefined> {
                     <button
```

Comment:
We need to remove the button tag, the React Router Link component render an anchor html element and not need to be surrouned by a button tag. The button element doesn't even allow to trigger the button action with space keydown/keyup.

Reply:
Okay an anchor wrapped inside a button allows to disable the link anchor.